frameworks/helloworld/src/main/dist/multiport.yml

```diff
@@ -0,0 +1,65 @@
+name: {{FRAMEWORK_NAME}}
+scheduler:
+  principal: {{FRAMEWORK_PRINCIPAL}}
+  user: {{FRAMEWORK_USER}}
+pods:
+  multiport:
+    count: {{HELLO_COUNT}}
+    resource-sets:
+      multi-port-resources:
+        ports:
+          port_one:
+            port: {{HELLO_PORT_ONE}}
+          port_two:
+            port: {{HELLO_PORT_TWO}}
+          port_three:
```

Comment:
Does one need three ports, or are two sufficient?

Reply:
Two should be enough. The PR is still a `wip` and i was testing something else.